edxval/models.py

```diff
@@ -417,6 +417,28 @@ class VideoTranscript(TimeStampedModel):
     class Meta:
         unique_together = ('video_id', 'language_code')
 
+    @property
+    def filename(self):
+        """
+        Returns readable filename for a transcript
+        """
+        try:
+            video = Video.objects.get(edx_video_id=self.video_id)
+            client_id, __ = os.path.splitext(video.client_video_id)
+            file_name = u'{name}-{language}.{format}'.format(
+                name=client_id,
```

Comment:
Should we reduce file name length ?

Reply:
client_video_id is primarily the file name of an uploaded video, so we don't need to do this.